test/test_parser.cpp

```diff
@@ -1340,11 +1351,14 @@ TEST(Parser_substitution)
     verify_query_sub(test_context, t, "(age > $0 || fees == $1) && age == $0", args, num_args, 1);
 
     // negative index
-    CHECK_THROW_ANY(verify_query_sub(test_context, t, "age > $-1", args, num_args, 0));
+    // FIXME: Should the error be std::out_of_range or SyntaxError?
```

Comment:
I don't have strong opinions on these, I can see arguments for either so I'd just go with whatever it currently is.